launcher/InstanceCopyPrefs.h

```diff
@@ -0,0 +1,41 @@
+//
+// Created by marcelohdez on 10/22/22.
+//
+
+#pragma once
+
+#include <QStringList>
+
+struct InstanceCopyPrefs {
+   public:
+    [[nodiscard]] bool allTrue() const;
+    [[nodiscard]] QString getSelectedFiltersAsRegex() const;
+    // Getters
+    [[nodiscard]] bool isCopySavesEnabled() const;
+    [[nodiscard]] bool isKeepPlaytimeEnabled() const;
+    [[nodiscard]] bool isCopyGameOptionsEnabled() const;
+    [[nodiscard]] bool isCopyResourcePacksEnabled() const;
+    [[nodiscard]] bool isCopyShaderPacksEnabled() const;
+    [[nodiscard]] bool isCopyServersEnabled() const;
+    [[nodiscard]] bool isCopyModsEnabled() const;
+    [[nodiscard]] bool isCopyScreenshotsEnabled() const;
+    // Setters
+    void enableCopySaves(bool b);
+    void enableKeepPlaytime(bool b);
+    void enableCopyGameOptions(bool b);
+    void enableCopyResourcePacks(bool b);
+    void enableCopyShaderPacks(bool b);
+    void enableCopyServers(bool b);
+    void enableCopyMods(bool b);
+    void enableCopyScreenshots(bool b);
```

Comment:
I don't think we need getters and setters here. As it's just a struct, we can make the fields public. Should reduce the code a bit.

Reply:
heh i asked them to make getters and setters :p

having them should make it easier to change the struct internals if we ever need to

Reply:
> having them should make it easier to change the struct internals if we ever need to

Fair, but I don't know if we would ever change the internal much though :eyes:

Reply:
better safe than sorry :p ~i need to somehow justify the work i made them do~

Reply:
I guess it's fine then. Can't promise that I would never remove this during a refactor though :eyes: